portfolio/src/main/webapp/index.html

```diff
@@ -10,8 +10,41 @@
         <div id="content">
             <h1>My Portfolio</h1>
             <h2>This is Ankita's portfolio.</h2>
+
+            <p2>
+                <div style="text-align:center;padding:1em 0;"> <h4>
+                    <a style="text-decoration:none;" href="https://www.zeitverschiebung.net/en/city/5368361">
+                    <span style="color:gray;">Current local time in</span><br />Los Angeles, United States</a></h4> 
+                    <iframe id = "time" src="https://www.zeitverschiebung.net/clock-widget-iframe-v2?language=en&size=small&timezone=America%2FLos_Angeles" 
```

Comment:
This is pretty cool :)

Reply:
Unfortunately, the site I'm embedding from is down right now :( 
Maybe I can add a feature at some point to display a better error message if the site is down!

Reply:
That's a good idea.